documentation/modules/ROOT/pages/basics.adoc

```diff
@@ -4,19 +4,45 @@
 
 == Creating a Quarkus Application
 
+[tabs]
+====
+Maven::
++ 
+--
 [.console-input]
 [source,bash,subs="+macros,+attributes"]
 ----
 mvn "io.quarkus:quarkus-maven-plugin:create" \
   -DprojectGroupId="com.redhat.developers" \
   -DprojectArtifactId="{project-name}" \
   -DprojectVersion="1.0-SNAPSHOT" \
-  -DclassName="HelloResource" \
+  -DclassName="GreetingResource" \
```

Comment:
Renamed this to the default "GreetingResource" which is added when you create a new app with Quarkus CLI.